lib/ansible/plugins/test/link_exists.yml

```diff
@@ -0,0 +1,21 @@
+DOCUMENTATION:
+  name: link_exists
+  author: Ansible Core
+  version_added: "2.5"
+  short_description: does the path exist, no follow
+  description:
+    - Check if the provided path maps to an existing symlink on the controller's filesystem (localhost)
+    - Does not Follow symlinks, so it only verifies that the link itself exists.
+  options:
+    _input:
+      description: a path
+      type: path
+
+EXAMPLES:
+    ismyhostsalink: "{{ '/etc/hosts' is link_exists}}"
+    list_of_symlinks: "{{ list_of_paths | select('link_exists') }}"
+
+RETURN:
+  _value:
+    description: Returns C(True) the path corresponds to an existing filesystem object on the controller, c(False) if otherwise
```

Comment:
```suggestion
    description: Returns C(True) if the path corresponds to an existing filesystem object on the controller, C(False) if otherwise.
```